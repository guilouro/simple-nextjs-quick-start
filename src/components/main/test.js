import React from 'react'
import Main from '.'
import { render } from 'react-testing-library'

describe('Main', () => {
  it('Should render correctly', () => {
    const { container } = render(<Main />)
    expect(container).toMatchSnapshot()
  })
})
