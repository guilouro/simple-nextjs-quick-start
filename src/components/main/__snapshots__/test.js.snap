// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main Should render correctly 1`] = `
<div>
  <h1
    class="sc-bdVaJa khUSez"
  >
    Welcome
  </h1>
</div>
`;
